torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -4,8 +4,8 @@
 import subprocess
 
 from .constants import (API_BLAS, API_C10, API_CAFFE2, API_DRIVER, API_FFT,
-                        API_PYTORCH, API_RAND, API_ROCTX, API_RTC, API_RUNTIME,
-                        API_SPARSE, CONV_CACHE, CONV_CONTEXT, CONV_D3D9,
+                        API_PYTORCH, API_PYT_EXT, API_RAND, API_ROCTX, API_RTC, API_RUNTIME,
```

Comment:
@alugorey Not sure where this crept in from, but lint error seems valid?

Reply:
Ah shoot. I fixed this during the rebase but forgot to mark it via git add so it wasn't added.